### PR TITLE
Mn

### PR DESCRIPTION
**Pull request by lax241**
Add spring-gateway-ocd

### CODE REVIEW

File: spring-gateway-ocd/README.md [hunk 1]
```diff
@@ -0,0 +1,12 @@
+Running Spring Gateway OCR
```
Comment (kospiotr) [inline spring-gateway-ocd/README.md]: OCD states for one click deployment. Please use just spring-gateway instead spring-gateway-ocd

File: spring-gateway-ocd/dev/pom.xml [hunk 2]
```diff
@@ -0,0 +1,113 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+	<modelVersion>4.0.0</modelVersion>
+
+	<groupId>com.gft</groupId>
+	<artifactId>spring-gateway-ocd</artifactId>
```
Comment (kospiotr) [inline spring-gateway-ocd/dev/pom.xml]: Remove OCD from  here

File: spring-gateway-ocd/dev/src/main/resources/application.yml [hunk 3]
```diff
@@ -0,0 +1,37 @@
+spring:
+  application:
+    name: spring-gateway-ocd
```
Comment (kospiotr) [inline spring-gateway-ocd/dev/src/main/resources/application.yml]: Remove OCD

**Comment by lax241** [comment]
Now should be good.

**Pull request merged**